src/core/tsi/s2a/handshaker/s2a_handshaker_client.cc

```diff
@@ -235,15 +259,15 @@ tsi_result s2a_handshaker_client_create(
     grpc_pollset_set* interested_parties,
     const grpc_s2a_credentials_options* options, const grpc_slice& target_name,
     grpc_iomgr_cb_func grpc_cb, tsi_handshaker_on_next_done_cb cb,
-    void* user_data, bool is_client, S2AHandshakerClient** client) {
+    void* user_data, bool is_client, bool is_test, S2AHandshakerClient** client) {
```

Comment:
Please also change the function name to adhere to C++ naming convention.

Reply:
Done in next commit (and same for the destroy method).